jakelib/gallery.jake

```diff
@@ -0,0 +1,30 @@
+const { im, yaml } = require('./utils')
+const exec = require('mz/child_process').exec
+
+const gallery = yaml('_data/gallery.yml')
+  .map(x => { x.name = x.image.replace(/\..*$/, ''); return x })
+
+const galleryDir = 'assets/images/gallery'
+
+directory(galleryDir)
+
+gallery.forEach(item => {
+  const image = `${galleryDir}/${item.image}`
+
+  file(image, [galleryDir, 'screenshot'], async () => {
```

Comment:
Did you know it's been [in core](https://github.com/SassDoc/sassdoc/blob/master/src/sassdoc.js#L129) since 2.0 ? ;)